src/index.js

```diff
@@ -4,7 +4,7 @@ import App from "./components/App.jsx";
 import { Provider } from 'react-redux';
 import {store} from './store.js';
 
-ReactDOM.render(
+ReactDOM.hydrate(
```

Comment:
What is .hydrate doing here? (just out of curiosity)

Reply:
@nastassiafulconis my bad this was when i was testing server side rendering! ill update this pr and sorry im getting back to you so late!